src/doc.cpp

```diff
@@ -888,10 +888,12 @@ void Doc::PrepareData()
     this->Process(prepareLayerElementParts);
 
     /************ Add default syl for syllables (if applicable) ************/
-    ListOfObjects syllables = this->FindAllDescendantsByType(SYLLABLE);
-    for (Object *object : syllables) {
-        Syllable *syllable = dynamic_cast<Syllable *>(object);
-        syllable->MarkupAddSyl();
+    if (!this->IsTranscription()) {
+        ListOfObjects syllables = this->FindAllDescendantsByType(SYLLABLE);
+        for (Object *object : syllables) {
+            Syllable *syllable = dynamic_cast<Syllable *>(object);
+            syllable->MarkupAddSyl();
+        }
```

Comment:
I am pretty sure this was added for Neon (see https://github.com/rism-digital/verovio/commit/7beaab7a83dd4977c71d19bbb3ddf6a65dced681), so if you do not need it, just remove it completely.